tests/bluetooth/tester/overlay-bt_ll_sw_split.conf

```diff
@@ -38,6 +38,11 @@ CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT=3
 CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
 CONFIG_BT_ISO_MAX_BIG=2
 
+# Match supported periodic advertising ext adv instances
+CONFIG_BT_CTLR_ADV_SYNC_SET=2
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=6
```

Comment:
Previously we 1 have ADV_SYNC_SET and BT_CTLR_ADV_DATA_BUF_MAX=2, but now we triple the buffers because we double of PA sets?

Reply:
> No, CONFIG_BT_CTLR_ADV_DATA_BUF_MAX had to be increased to handle assert reported in https://github.com/zephyrproject-rtos/zephyr/issues/92153

But it worked with `CONFIG_BT_CTLR_ADV_SYNC_SET=1` and `CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=2`, but not `CONFIG_BT_CTLR_ADV_SYNC_SET=2` and `CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=4`

Shouldn't it scale linearly?

Reply:
> > No, CONFIG_BT_CTLR_ADV_DATA_BUF_MAX had to be increased to handle assert reported in #92153
> 
> But it worked with `CONFIG_BT_CTLR_ADV_SYNC_SET=1` and `CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=2`, but not `CONFIG_BT_CTLR_ADV_SYNC_SET=2` and `CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=4`
> 
> Shouldn't it scale linearly?

don't know,  maybe host (or tester?) should be able to reuse one instance (currently it first use instance 0 and then setup instance 1)

Reply:
CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=4  is enough, I'll update PR